extensions/ql-vscode/src/variant-analysis/gh-api/gh-api-client.ts

```diff
@@ -7,6 +7,39 @@ import {
   VariantAnalysisSubmissionRequest,
 } from "./variant-analysis";
 import { Repository } from "./repository";
+import { Progress } from "vscode";
+import { CancellationToken } from "vscode-jsonrpc";
+
+export async function getCodeSearchRepositories(
+  credentials: Credentials,
+  query: string,
+  progress: Progress<{
+    message?: string | undefined;
+    increment?: number | undefined;
+  }>,
+  token: CancellationToken,
+): Promise<string[]> {
+  let nwos: string[] = [];
```

Comment:
I think this can be a const.

Reply:
Line 36 starts complaining because I 'reassign' there it seems. Guess there would be a different solution with slice() for example but 🤷‍♀️

Reply:
Ah yeah I missed that line. What you've got makes sense!